upd_script/antenna_wifi_override.conf

```diff
@@ -0,0 +1,9 @@
+[Unit]
+[Unit]
```

Comment:
Take out one `[Unit]`. It's redundant having 2 of them.

Reply:
woah, how did that extra one get in??

Reply:
sneaky VS code was hiding line 1 from me.  Fixed now.